test/integration/openid4ci_test.go

```diff
@@ -209,4 +223,16 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 		testHelper.CheckActivityLogAfterOpenID4CIFlow(t, vcsAPIDirectURL, tc.issuerProfileID, subID)
 		testHelper.CheckMetricsLoggerAfterOpenID4CIFlow(t, tc.issuerProfileID)
 	}
+
+	time.Sleep(5 * time.Second)
```

Comment:
Reason for the `time.Sleep`? These will slow down our tests

Reply:
Can these be replaced with retries?

Reply:
logs will not appear on the traces collector immediately, some timeout needed to wait until it will process them

Reply:
Sounds good